modin/engines/base/frame/data.py

```diff
@@ -1310,6 +1310,7 @@ def _apply_full_axis(
         new_index=None,
         new_columns=None,
         dtypes=None,
+        enumerate_partitions=False,
```

Comment:
I am generally against adding more arguments to the internal APIs. Is there no other way?

Reply:
I'll look for another way.